frontend/pages/approve.tsx

```diff
@@ -1,34 +1,25 @@
 import type { NextPage } from 'next'
 import Layout from '../components/Layout'
 import SEO from '../components/SEO'
-import {
-  useAnchorWallet,
-  useConnection,
-  useWallet,
-} from '@solana/wallet-adapter-react'
-import {
-  PythBalance,
-  StakeAccount,
-  StakeConnection,
-  STAKING_ADDRESS,
-} from '@pythnetwork/staking'
+import { useAnchorWallet } from '@solana/wallet-adapter-react'
+import { StakeAccount } from '@pythnetwork/staking'
 import { useEffect, useState } from 'react'
-import { Wallet } from '@coral-xyz/anchor'
 import { PublicKey } from '@solana/web3.js'
 import { useRouter } from 'next/router'
 import toast from 'react-hot-toast'
 import { capitalizeFirstLetter } from 'utils/capitalizeFirstLetter'
+import { useStakeConnection } from 'hooks/useStakeConnection'
+import { useSplitRequest } from 'hooks/useSplitRequest'
 
 const ApproveSplit: NextPage = () => {
-  const { connection } = useConnection()
   const anchorWallet = useAnchorWallet()
-  const [amount, setAmount] = useState<PythBalance>()
-  const [recipient, setRecipient] = useState<PublicKey>()
 
-  const [stakeConnection, setStakeConnection] = useState<StakeConnection>()
   const [stakeAccounts, setStakeAccounts] = useState<StakeAccount[]>()
```

Comment:
I can't use useStakeAccounts here because I don't want the accounts of the connected wallet here.
I want the accounts of an arbitrary wallet.